libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/incident/IncidentInfo.kt

```diff
@@ -34,7 +45,13 @@ class IncidentInfo internal constructor(
     val description: String?,
     val subType: String?,
     val subTypeDescription: String?,
-    val alertcCodes: List<Int>?
+    val alertcCodes: List<Int>?,
+    val countryCodeAlpha2: String?,
+    val countryCodeAlpha3: String?,
+    val lanesBlocked: List<String>,
```

Comment:
should be `StringDef`?

Reply:
not sure if NN has a final list of possible values, we can add it later when NN tests this logic